saleor/extensions/plugins/webhook/tasks.py

```diff
@@ -16,7 +16,7 @@
 @app.task
 def trigger_webhooks_for_event(event_type, data):
     permissions = {}
-    required_permission = WebhookEventType.PERMISSIONS[event_type]
+    required_permission = WebhookEventType.PERMISSIONS[event_type].value
```

Comment:
Have you checked that? It doesn't look like it will work. 
Especially `app_label, codename = required_permission.split(".")`

Reply:
`required_permission` will be like <content_type>.<permission_codename>

`WebhookEventType.PERMISSIONS[event_type]` returns Enum